src/components/organisms/Footer.tsx

```diff
@@ -4,7 +4,9 @@ import { Images } from '../../images/Images'
 
 import './Footer.scss'
 
-export class Footer extends React.Component<undefined, undefined> {
+interface FooterProps {}
```

Comment:
Is this change needed for the version upgrade?

Reply:
yes